source/includes/inkling-reference/_lessons.html.md

```diff
@@ -194,15 +359,29 @@ relOp ::=
   '==' | '<' | '>' | '<=' | '>='
 ```
 
-Select the Syntax tab for the **until** clause syntax.
+Future support of the `until` clause will include `minimize` and constant
+expressions. Current support is limited to `maximize`.
+
+### Usage
 
-The **until** clause is required if the curriculum _trainingSpecifier_ is **simulator**. If it is not present, a default with value minimize will be created.
+The `until` clause is required.
 
-The **until** clause in the lesson specifies the termination condition for training. The **until** clause in our breakout example above was this:
+### Example
 
-  *until minimize ball_location_distance*
+There are several examples of the `until` clause above. They are excerpted in
+the panel to the left.
```

Comment:
I would say "code panel" since the panel is to the right on large screens and above the text on small screens.